commons/src/main/java/io/hyscale/commons/exception/CommonErrorCode.java

```diff
@@ -16,6 +16,8 @@
 package io.hyscale.commons.exception;
 
 public enum CommonErrorCode implements HyscaleErrorCode {
+    FAILED_TO_GET_USER_INPUT("Failed to get user input"),
+    INVALID_INPUT_BY_USER("User provided invalid input {}"),
```

Comment:
Can you think of a better error message. Refering "User" is not so great in the error message.